Vaadin 25: Compilation failure (package javax.annotation does not exist)

### Describe the bug

[ERROR] MenuItemDto.java:[25,24] package javax.annotation does not exist
[ERROR] MenuItemDto.java:[30,6] cannot find symbol
[ERROR]   symbol:   class Nullable
[ERROR]   location: class com.flowingcode.addons.applayout.endpoint.MenuItemDto

### Expected behavior

mvn clean compile -Pv25

### Minimal reproducible example

_No response_

### Add-on Version

6.1.1-SNAPSHOT

### Vaadin Version

25.0.0

### Additional information

_No response_

--- a/pom.xml
+++ b/pom.xml
@@ -534,7 +534,7 @@
 			<properties>
 				<maven.compiler.source>21</maven.compiler.source>
 				<maven.compiler.target>21</maven.compiler.target>
-				<vaadin.version>25.0.0-beta2</vaadin.version>
+				<vaadin.version>25.0.0</vaadin.version>
 				<jetty.version>11.0.26</jetty.version>
 			</properties>
 			<dependencies>

--- a/pom.xml
+++ b/pom.xml
@@ -75,10 +75,6 @@
             <id>Vaadin Directory</id>
             <url>https://maven.vaadin.com/vaadin-addons</url>
         </repository>
-        <repository>
-			<id>FlowingCode Releases</id>
-			<url>https://maven.flowingcode.com/releases</url>
-		</repository>
 		<repository>
 			<id>FlowingCode Snapshots</id>
 			<url>https://maven.flowingcode.com/snapshots</url>

--- a/pom.xml
+++ b/pom.xml
@@ -4,7 +4,7 @@
 
     <groupId>com.flowingcode.addons.applayout</groupId>
     <artifactId>app-layout-addon</artifactId>
-    <version>6.1.1-SNAPSHOT</version>
+    <version>6.2.0-SNAPSHOT</version>
     <name>App Layout Addon</name>
     <description>Integration of app-layout for Vaadin Flow</description>
 

--- a/src/main/java/com/flowingcode/addons/applayout/endpoint/MenuItemDto.java
+++ b/src/main/java/com/flowingcode/addons/applayout/endpoint/MenuItemDto.java
@@ -2,7 +2,7 @@
  * #%L
  * App Layout Addon
  * %%
- * Copyright (C) 2018 - 2022 Flowing Code
+ * Copyright (C) 2018 - 2025 Flowing Code
  * %%
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
@@ -22,7 +22,7 @@ package com.flowingcode.addons.applayout.endpoint;
 import com.vaadin.hilla.Nonnull;
 import java.util.ArrayList;
 import java.util.List;
-import javax.annotation.Nullable;
+import jakarta.annotation.Nullable;
 
 public class MenuItemDto {
 

